js/data.js

```diff
@@ -6,3 +6,13 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+var previousEntriesJSON = localStorage.getItem('entries');
+if (previousEntriesJSON !== null) {
+  previousEntriesJSON = JSON.parse(data);
+}
+
+window.addEventListener('beforeunload', function (event) {
+  var savedData = JSON.stringify(data);
+  localStorage.setItem('Entries', savedData);
```

Comment:
make sure the case of the property you create is matching the one you are accessing!